adds support for building dust to html files through html-webpack-plugin

RobertDaleSmith opened the pull request:
👋 Hi! I like this loader but I wanted to be able to also render out static html files from my dust templates with partials. I attempted to use dust-loader-complete with html-webpack-plugin but I kept hitting errors. Just seems the plugin doesn't know how to handle the compiled dust scripts. So I added an option flag and a few tweaks to dust-loader-complete get it working. I wanted to maintain normal output for client-side compiled dust templates and found I could set the loader inline as shown in the example.

```
plugins: [
   new HtmlWebpackPlugin({
      filename: 'dust_test2/index.html',
      template: '!!dust-loader-complete?htmlOutput=true!src/views/test.dust',
   }),
],
```

Not sure I handled this the best way but everything is working how I wanted it to. 😁🤞 I figured I would share for feedback and to contribute if its something you guys fits with your package.

jrrbru commented:
Hi there! Thanks for contributing. At first glance this looks good. Could I ask you to take add a test of the new option to the unit tests?

RobertDaleSmith commented:
@jrrbru I added the new option to the unit tests. Also for simplification I changed the option name to `render`. And I added details to the readme. Any changes needed that you can think of?
